src/Api/Helpers/FriendlyExceptionsMiddleware.cs

```diff
@@ -21,10 +22,17 @@ public Task InvokeAsync(HttpContext context)
 
     private async Task SafeNext(HttpContext context)
     {
+        var env = context.RequestServices.GetService<IWebHostEnvironment>();
+
         try
         {
             await _next(context);
         }
+        catch (SqliteException e) when (env.IsDevelopment())
+        {
+            // this is done to allow migrations to be applied in development mode
+            throw;
+        }

```

Comment:
Very nice, I tried to make the migrations endpoint in Api to but this was the solution I didn't think of, it was always being grabbed by our safe one. This doesn't work for when you use MS SQL for development but we could add a similar exception but I think that's fine for now. Sqlite will be the default dev DB so this is great.